brozzler/cli.py

```diff
@@ -544,12 +544,25 @@ def dump_state(signum, frame):
         finally:
             signal.signal(signal.SIGQUIT, dump_state)
 
+    def get_skip_av_seeds():
+        skip_av_seeds_file = "/opt/local/brozzler/skip_av_seeds.txt"
```

Comment:
nit: This is a constant, so it should be upper cased.

Reply:
thanks, updated